### DISCUSSION TITLE
pve post install script

### DISCUSSION THREAD

**Discussion by ehinkle27** [tteck/Proxmox#2944] [Help Needed] [answered]
Hi I noticed in the pve post install script for the ceph repositories everything is commented out, should only the enterprise subscriptions get commented out?
```
cat <<EOF >/etc/apt/sources.list.d/ceph.list
# deb http://download.proxmox.com/debian/ceph-quincy bookworm enterprise
# deb http://download.proxmox.com/debian/ceph-quincy bookworm no-subscription
# deb http://download.proxmox.com/debian/ceph-reef bookworm enterprise
# deb http://download.proxmox.com/debian/ceph-reef bookworm no-subscription
EOF
```

**Comment by tteck** [comment, marked as answer]
You enable the one you want to utilize in the Proxmox GUI

**Reply by tteck** [reply]
![Screenshot 2024-05-09 9 28 49 AM](https://github.com/tteck/Proxmox/assets/21342670/bed1c0ba-cd19-4388-afa3-10c91278372c)
